python/grass/jupyter/utils.py

```diff
@@ -0,0 +1,127 @@
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains utility functions for InteractiveMap.
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#            This program is free software under the GNU Gernal Public
+#            License (>=v2). Read teh file COPYING that comes with GRASS
```

Comment:
typo: GNU **Gernal** Public...Read **teh** file COPYING that comes with GRASS

Same in python/grass/jupyter/interact_display.py.

You should not be writing these by hand.

Reply:
Ah, good catch. I went hunting for the proper header to put here and found some conflicting methods. There's the [https://trac.osgeo.org/grass/wiki/Submitting/Python](guidelines on the wiki) which are similar to what I have but as a docstring instead of a comment. But there's also lots (most?) of python scripts on github that use comments (see [https://github.com/OSGeo/grass/blob/master/python/grass/utils/download.py](utils/download.py) or [https://github.com/OSGeo/grass/blob/master/lib/init/grass.py](lib/init/grass.py)). There's also examples I found in the python directory that have more a Sphinx docstring style (such as [https://github.com/OSGeo/grass/blob/master/python/grass/script/raster.py](raster.py)). What do you think is most appropriate here?

Reply:
This style of header is fine. Just don't write it by hand and if you do, don't include typos :-)

Reply:
Typos fixed. I'll copy and paste next time 👍